### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by Arturgouveia1970**
1- Moved README file to the main directory.
2- Added and deleted information to README file.
3- corrected linter errors.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
+test.md
```
Comment (Omar-Muhamad) [inline .gitignore:1]: - [ ] Kindly add `node_modules` to you `.gitignore ` file.

**Review by Omar-Muhamad — COMMENTED** [review]
# Changes Required ♻️
### Hi @Arturgouveia1970 

Good job so far, you are almost there.



## (Highlights) Good Points: :+1:
- Github flow is used correctly. :heavy_check_mark:
- You documented your work professionally. :heavy_check_mark:
- You added .html and .css files the right way. :heavy_check_mark:

## (Changes Required) Aspects to improve: ♻️

- [ ] Please don't close or merge the Pull Request until you got the approval from Code reviewer so kindly open this Pull Request again and implement the following requirements. 

- [ ]  Kindy make sure all linter result checked green. AKA: Linter should report NO errors so please fix Styelint errors.

- [ ] Kindly make sure the Pull Request have a descriptive title telling you what changes are inside so please edit it.

- [ ] Please make sure the Pull Request have a summary describing introduced changes in more detail so kindly add a description.

- [ ] Kindly move README file in the main directory of your repo instead of readme-template-master directory 

- [ ] In README file please make sure that you correct the data inside These sections (Project name, project description, Getting started, Built with and please add social links inside author section).

- [ ] In README file kindly remove unused sections like (Prerequisites, Setup, Install, etc.) and author 2 because you don't have a collaborator in this project.

- [ ] Nice work adding a meaningful commit messages but next time please capitalize the first letter of your message.

- [ ] Please check the inline comment below ⬇.

## [Optional] suggestions:

- [Optional] In the Pull Request description it would be better to add every change you did in checklist or bullet points.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Arturgouveia1970** [comment]
reopening pull-request

File: README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+- Html
+- css
+
+## Authors
+
+👤 **Artur Gouveia**
+
+- GitHub:GitHub: [@Arturgouveia1970](https://github.com/Arturgouveia1970)
+
```
Comment (Omar-Muhamad) [inline README.md:17]: [Optional] Please add the rest of Your social links.

**Review by Omar-Muhamad — APPROVED** [review]
Hi @Arturgouveia1970 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## [Optional] suggestions:

- [Optional] Kindly next time add `Getting Started` to your readme file to make your repo more professional.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Arturgouveia1970** [comment]
Thank you, Omar. Enviado do meu telemóvel Huawei.-------- Mensagem original --------De: Omar Muhammad ***@***.***>Data: terça-feira, 15 de março de 2022, 21:10Para: Arturgouveia1970/Hello-world ***@***.***>Cc: Arturgouveia1970 ***@***.***>, Mention ***@***.***>Assunto: Re: [Arturgouveia1970/Hello-world] Hello Microverse Project (PR #1)

@Omar-Muhamad approved this pull request.



Hi @Arturgouveia1970 ,

Your project is complete!  There is nothing else to say other than... it's time to merge it 

Congratulations! 🎉



[Optional] suggestions:

[Optional] Kindly next time add Getting Started to your readme file to make your repo more professional.

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.



As described in the Code reviews limits policy  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.



In README.md:

> +- GitHub:GitHub: ***@***.***(https://github.com/Arturgouveia1970)

+



[Optional] Please add the rest of Your social links.

**Pull request merged**